lib/magma/attributes/file.rb

```diff
@@ -1,8 +1,9 @@
 class Magma
   class FileAttribute < Attribute
     def initialize(name, model, opts)
-      super
       @type = String
+      Magma.instance.storage.setup_uploader(model, name, :file) 
```

Comment:
ImageAttribute inherits from this method, but does not replace this setup_uploader call with one using the :image argument. I think the uploading code is poorly-tested, so this might have slipped under the radar.

Reply:
I'm not sure how to fix this? I can add an initialize method to ImageAttribute, but there's nothing stop this from being run in the ImageAttribute#new and then again in FileAttribute#new when super is called?